### PR TITLE
SpriteInfo - Design pass, updated SVG icons

### LINKED ISSUE #88: Set order for tabbing through sprite info groups/forms

### PR DESCRIPTION
**Pull request by lifeinchords**
* SpriteSelector: fixed bottom padding from scroll wrapper
* Added tab indexes, fixes #88

### CODE REVIEW

File: src/components/sprite-info/icon--hide.svg [hunk 1]
```diff
@@ -1 +1,13 @@
-<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 14 11"><defs><style>.cls-1{fill:none;stroke:#575e75;stroke-linecap:round;stroke-linejoin:round;}</style></defs><title>icon--hide</title><path class="cls-1" d="M13.5,5.5s-2.91,4-6.5,4S.5,5.5.5,5.5s2.91-4,6.5-4S13.5,5.5,13.5,5.5Z"/><circle class="cls-1" cx="7" cy="5.5" r="2"/><line class="cls-1" x1="2" y1="10.5" x2="12" y2="0.5"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (thisandagain) [inline src/components/sprite-info/icon--hide.svg]: This should be minified.

File: src/components/sprite-info/icon--x.svg [hunk 2]
```diff
@@ -1 +1,24 @@
-<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 11 13"><defs><style>.cls-1{opacity:0.8;}.cls-2,.cls-3{fill:none;stroke:#575e75;stroke-linecap:round;stroke-linejoin:round;}.cls-3{stroke-dasharray:0 2;}</style></defs><title>icon--x</title><g class="cls-1"><line class="cls-2" x1="0.5" y1="6.5" x2="10.5" y2="6.5"/><polyline class="cls-2" points="8.73 4.73 10.5 6.5 8.73 8.27"/><polyline class="cls-2" points="2.27 8.27 0.5 6.5 2.27 4.73"/><line class="cls-3" x1="5.5" y1="0.5" x2="5.5" y2="12.5"/></g></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (thisandagain) [inline src/components/sprite-info/icon--x.svg]: This should also be minified.

File: src/css/units.css [hunk 3]
```diff
@@ -3,5 +3,5 @@ $space: 0.5rem;
 $sprites-per-row: 5;
 
 $menu-bar-height: 3rem;
-$sprite-info-height: 5rem;
+$sprite-info-height: 5.25rem; /* TODO: SpriteInfo isn't explicitly set to this height yet */
```
Comment (thisandagain) [inline src/css/units.css]: Generally we use `@todo` which makes these types of notes easier to find.

Comment (lifeinchords) [inline src/css/units.css]: should I do a pass to update the convention? Added a lot of these in the last few weeks

Comment (thisandagain) [inline src/css/units.css]: That would be nice if you have time.

**Comment by lifeinchords** [comment]
@rschamp @thisandagain accidentally closed this earlier, if you have a few please have a look. Thanks @carljbowman for the heads up

File: src/components/sprite-info/sprite-info.css [hunk 4]
```diff
@@ -13,89 +16,91 @@
...
-    opacity: 0.4;
+    width: 100%;
+    height: 100%;
 }
 
-.icon.is-active {
-    opacity: 1;
-}
+.radio-box .icon { opacity: 0.4; }
+.radio-box .icon.is-active { opacity: 1; }
```
Comment (rschamp) [inline src/components/sprite-info/sprite-info.css]: Can this nesting be avoided?

Comment (lifeinchords) [inline src/components/sprite-info/sprite-info.css]: Yup, this was just for the playtest

File: src/components/sprite-info/sprite-info.css [hunk 5]
```diff
@@ -13,89 +16,98 @@
...
 
 .icon {
-    opacity: 0.4;
+    width: 100%;
+    height: 100%;
 }
 
-.icon.is-active {
-    opacity: 1;
+.show-icon,
```
Comment (lifeinchords) [inline src/components/sprite-info/sprite-info.css]: @rschamp is this good?

**Comment by lifeinchords** [comment]
@rschamp how about now? :)

**Pull request merged**